tests/unit/backend/wmg/test_query.py

```diff
@@ -128,240 +100,6 @@ def test__query_expression_summary_default_cube__returns_correct_results(self):
             expected = [553379.0, 261191.0]
```

Comment:
Why not move `test__query_expression_summary_default_cube__returns_correct_results` into `test_expression_dotplot`?

Reply:
@atarashansky - My thought was `test_query.py` tests methods in the `query.py` module which holds the `WMGQuery` class. Therefore I thought this test calling `WMGQuery.expression_summary_default()` should belong here.

What I hope we can do, is that the `unit` folder (holding the unit tests), reflects the directory hierarchy of `backend/wmg`. I would have moved `test_query.py` into the `test/unit/backend/wmg/data` folder but I wanted to do minimal change to accomplish the isolation that I need so as not to create too much churn in the codebase.

I will go ahead and merge this as is. And if you have a different idea of how to go about refactoring/reorganizing tests and code, let's discuss.